redis/utils.py

```diff
@@ -117,3 +118,12 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+logger = logging.getLogger("push_response")
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler()
+handler.setLevel(logging.INFO)
+logger.addHandler(handler)
+
+INFO_LOGGER = logger
```

Comment:
INFO_LOGGER Should instead be _info_logger, so that we don't get caught in the same issue as the parsers rename